.github/workflows/module-checks.yml

```diff
@@ -82,12 +82,16 @@ jobs:
             echo "language=python" >> "$GITHUB_OUTPUT"; \
           elif [ -f $MODULE_PATH/pyproject.toml ]; then \
             echo "language=python" >> "$GITHUB_OUTPUT"; \
+          else \
+            echo "language=none" >> "$GITHUB_OUTPUT"; \
```

Comment:
Alternately we could run something like cfn-lint if we find a `modulestack.yaml`? There are some modules which have both Python + `modulestack.yaml` though.

Reply:
Yea can't hurt to add as a separate check outside this specific condition, no need in this PR though.